index.js

```diff
@@ -20,16 +25,23 @@ exports.start = function (options, _onStarted) {
   let directories = options.directories || [directory]
   let file = options.file || FILE
   let host = options.host || HOST
+  let loggerOptions = options.logger || LOGGEROPTIONS
   let onStarted = _onStarted || function () {}
 
   app.use(compression())
 
-  // First, check the file system
+  // configure logger
```

Comment:
Should probably only use the logger if they provide a config for it (not default to all the time)